dataverse-persistence/src/main/resources/db/migration/V44__addTablesForROR.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE IF NOT EXISTS rordata (
+    id BIGSERIAL PRIMARY KEY,
+    rorid CHAR(9) UNIQUE,
+    name VARCHAR,
+    countryname VARCHAR,
+    countrycode VARCHAR(16)
```

Comment:
same as above

Reply:
Also if any of them cannot be nullable I would also attach this information, since it is very useful